src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -8929,6 +8929,49 @@ public static int Main()
 
         #region SkipLocalsInitAttribute
 
+        [Fact]
+        public void SkipLocalsInitRequiresUnsafe()
+        {
+            var source = @"
+using System.Runtime.CompilerServices;
+
+[module: SkipLocalsInitAttribute]
+
+namespace System.Runtime.CompilerServices
+{
+    public class SkipLocalsInitAttribute : System.Attribute
+    {
+    }
+}
+
+[SkipLocalsInitAttribute]
+public class C
+{
+    [SkipLocalsInitAttribute]
+    public void M()
+    { }
+
+    [SkipLocalsInitAttribute]
+    public int P => 0;
+}
```

Comment:
Is this attribute supported on events?

Reply:
In the event of that we should add a test to cover ... 

---
In reply to: [325397161](https://github.com/dotnet/roslyn/pull/38709#discussion_r325397161) [](ancestors = 325397161)